doc/developer-guides/hld/hld-emulated-devices.rst

```diff
@@ -15,3 +15,4 @@ documented in this section.
    UART virtualization <uart-virt-hld>
    Watchdoc virtualization <watchdog-hld>
    GVT-g GPU Virtualization <hld-APL_GVT-g>
+   Atkbdc emulation <atkbdc-virt-hld>
```

Comment:
I would suggest replacing this text by the one we settle on in the other document (title), e.g. "AT Keyboard Controller"